index.js

```diff
@@ -42,4 +39,37 @@ app.get('/listings', (req, res) => {
   })
 })
 
+app.get('/profile/:id', (req, res) => {
+  knex('users')
+    .where('id', Number(req.params.id))
```

Comment:
You don't need `Number` here. SQL is always a giant string, so `knex` can take a string `id` just fine.